api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -274,6 +274,32 @@ public OrderGroup getOrderGroupByUuid(String uuid) throws DAOException {
 	public OrderGroup getOrderGroupById(Integer orderGroupId) throws DAOException {
 		return (OrderGroup) sessionFactory.getCurrentSession().get(OrderGroup.class, orderGroupId);
 	}
+
+	/**
+	 * @see OrderDAO#getOrderGroupsByPatient(Patient)
+	 */
+	@Override
+	public List<OrderGroup> getOrderGroupsByPatient(Patient patient) throws APIException {
+		Criteria criteria = sessionFactory.getCurrentSession().createCriteria(OrderGroup.class);
+		if (patient == null) {
```

Comment:
This is trivial but maybe the if statement belongs above the createCriteria line.

Reply:
good catch